packages/fluentui/docs/src/views/IconViewer.tsx

```diff
@@ -0,0 +1,239 @@
+  const { QnaIcon } = exports;
+  return (
+    <Card variables={{ borderColor: '#f2f2f2' }} style={{ background: 'white' }} {...props}>
+      <Card.Header>
+        <Flex gap="gap.small">
+          <QnaIcon size="largest" />
+          <Flex column>
+            <Text size="large" content="How to use the icons?" weight="bold" />
+            <Text
+              content={
+                <span>
+                  Just add the <code>@fluentui/react-icons-northstar</code> package and choose from over 200
+                  customizable icons.
+                </span>
+              }
+            />
+          </Flex>
+        </Flex>
+      </Card.Header>
+      <Card.Body>
+        <CodeSnippet mode="jsx" value={exampleCode} />
+      </Card.Body>
+    </Card>
+  );
+};
+
+const PlaygroundCard = props => {
+  const { QnaIcon, EditIcon } = exports;
+
+  const [bordered, setBordered] = React.useState(false);
+  const [circular, setCircular] = React.useState(false);
+  const [outline, setOutline] = React.useState(false);
+  const [rotate, setRotate] = React.useState(0);
+  const [size, setSize] = React.useState('largest');
+
+  return (
+    <Card variables={{ borderColor: '#f2f2f2' }} style={{ background: 'white', overflow: 'unset' }} {...props}>
+      <Card.Header>
+        <Flex gap="gap.small">
+          <EditIcon size="largest" />
+          <Flex column>
+            <Text size="large" content="Want to see different options on the icons?" weight="bold" />
+            <Text content="Use the controls below to see how the icons will change." />
+          </Flex>
+        </Flex>
+      </Card.Header>
+      <Card.Body>
+        <Flex>
+          <Flex column>
+            <div>
+              <Checkbox checked={outline} label="Outline" onChange={() => setOutline(!outline)} />
+            </div>
+            <div>
+              <Checkbox checked={bordered} label="Bordered" onChange={() => setBordered(!bordered)} />
+            </div>
+            <div>
+              <Checkbox checked={circular} label="Circular" onChange={() => setCircular(!circular)} />
+            </div>
+            <div>
+              <Text weight="semibold">Rotate</Text>
+              <br />
+              <Input
+                type="number"
+                placeholder="Enter angle of rotation"
+                value={rotate}
+                onChange={(e, data) => setRotate(Number(data.value))}
+              />
+            </div>
+            <div style={{ marginBottom: '7px' }}>
+              <Text weight="semibold">Size</Text>
+              <Dropdown
+                items={['smallest', 'smaller', 'small', 'medium', 'large', 'larger', 'largest']}
+                value={size}
+                onChange={(e, d) => setSize(d.value.toString())}
+              />
+            </div>
+            <Divider />
+            <div>
+              <Text>
+                Checkout our <Link to="components/svg-icon/definition">icon</Link> docs for more examples!
```

Comment:
Let's add some styling there, it's not clear that it's a link :(

![image](https://user-images.githubusercontent.com/14183168/79220005-8a8b6d00-7e53-11ea-8ee5-13f89629570e.png)